### PR TITLE
Update self-hosted-subscriptions.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
- Additional content for true-up process/screenshots.
- Added an FAQ entry

### CODE REVIEW

File: source/about/self-hosted-subscriptions.rst [hunk 1]
```diff
@@ -82,17 +82,27 @@ This process adds additional users to your existing subscription and is not a ne
...
+
+.. tip:: 
+   
+   Not sure where to take the screenshot? Please reach out to your Account Executive, CSM, orders@mattermost.com, or support@mattermost.com for help.
 
 .. image:: ../images/Site_Statistics.png
 	:alt: Take a screenshot from the System Console > Site Statistics page.
 
-If you have more total active users than you purchased in your annual subscription, we'll provide you with an invoice and an amendment to your purchase order for the new users added. The additional invoice will be pro-rated based on the number of months left in your subscription term, including the months for the calendar quarter for the time you pull the report. We will not provide downward adjustments. Mattermost will invoice based on Mattermost’s `current list prices <https://mattermost.com/pricing/>`__.
+If you have more total active users than you purchased in your annual subscription, your account executive will provide you with a true-up quote, invoice, and an amendment to your purchase order for the new users added.
```
Comment (johndavidlugtu) [inline source/about/self-hosted-subscriptions.rst:103]: "your Customer Success Manager will provide you with a true-up quote for the new users added."

- Remove mentions of invoice and amendment of purchase order
- deferring to you on whether to capitalize "Customer Success Representative" or not

File: source/about/self-hosted-subscriptions.rst [hunk 2]
```diff
@@ -82,17 +82,27 @@ This process adds additional users to your existing subscription and is not a ne
...
+When you buy an annual Mattermost subscription, you agree to provide quarterly reports of the actual number of registered users within your system. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
 
-We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric.
+We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us.
```
Comment (johndavidlugtu) [inline source/about/self-hosted-subscriptions.rst:90]: Can we incorporate this illustration for this?

Screenshot 2023-01-25 at 2 51 15 PM

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

File: source/about/self-hosted-subscriptions.rst (hunk 2 above)
Comment (justinegeffen) [reply source/about/self-hosted-subscriptions.rst:90]: We definitely can!

File: source/about/self-hosted-subscriptions.rst (hunk 1 above)
Comment (justinegeffen) [inline source/about/self-hosted-subscriptions.rst:103]:
```suggestion
If you have more total active users than you purchased in your annual subscription, your account executive will provide you with a true-up quote for the new users added.
```

File: source/about/self-hosted-subscriptions.rst [hunk 3]
```diff
@@ -82,17 +82,27 @@ This process adds additional users to your existing subscription and is not a ne
...
-We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric.
+We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. 
+
+- Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric. 
+- Please include the date of the screenshot in the file name.
+- We don't need your server address, so if it appears on your screenshot, you can redact it from the image.
+
+.. tip:: 
+   
+   Not sure where to take the screenshot? Please reach out to your Account Executive, CSM, orders@mattermost.com, or support@mattermost.com for help.
```
Comment (justinegeffen) [inline source/about/self-hosted-subscriptions.rst:98]:
```suggestion
   Not sure where to take the screenshot? Please reach out to your account executive, Customer Success Manager, orders@mattermost.com, or support@mattermost.com for help.
```

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

File: source/about/self-hosted-subscriptions.rst [hunk 4]
```diff
@@ -82,17 +82,29 @@ This process adds additional users to your existing subscription and is not a ne
 
 If you don't have a self-hosted license in the Customer Portal and want to add users, `contact us <https://mattermost.com/contact-us/>`__.
 
-Quarterly report
-^^^^^^^^^^^^^^^^
+Quarterly true-up report
+------------------------
 
-When you subscribe to an annual plan with Mattermost, you agree to provide quarterly reports of the actual number of registered users within your system if you're above the amount you subscribed in your purchase order. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
+When you buy an annual Mattermost subscription, you agree to provide quarterly reports of the actual number of registered users within your system. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
```
Comment (cwarnermm) [inline source/about/self-hosted-subscriptions.rst:88]: Consider changing formatting of "Inactive" from italics to bold, given that the docs are referencing the System Console UI.

**Review by cwarnermm — APPROVED** [review]
Reads well, @justinegeffen! Feedback provided inline. Approving for expediency.

**Review by cwarnermm — APPROVED** [review]

File: source/about/self-hosted-subscriptions.rst (hunk 4 above)
Comment (justinegeffen) [inline source/about/self-hosted-subscriptions.rst:88]:
```suggestion
When you buy an annual Mattermost subscription, you agree to provide quarterly reports of the actual number of registered users within your system. A registered user is a user who has an account in a workspace and does not show as **Inactive** in **System Console > User Management > Users**.
```

File: source/about/self-hosted-subscriptions.rst [hunk 5]
```diff
@@ -82,17 +82,29 @@ This process adds additional users to your existing subscription and is not a ne
...
+
+- Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric. 
+- Please include the date of the screenshot in the file name.
+- We don't need your server address, so if it appears on your screenshot, you can redact it from the image.
+
+.. tip:: 
+   
+   Not sure where to take the screenshot? Please reach out to your account executive, Customer Success Manager, orders@mattermost.com, or support@mattermost.com for help.
+   
+If you have more total active users than you purchased in your annual subscription, your account executive will provide you with a true-up quote for the new users added.
```
Comment (justinegeffen) [inline source/about/self-hosted-subscriptions.rst:105]:
```suggestion
If you have more total active users than you purchased in your annual subscription, your Customer Success Manager will provide you with a true-up quote for the new users added.
```

**Review by johndavidlugtu — APPROVED** [review]
Thanks Justine!

**Pull request merged**